src/Application/Components/Store/Store.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import StoreItem from "./StoreItem";
+
+const Store = React.createClass({
+  propTypes: {
+    assets: React.PropTypes.array.isRequired,
+    createOnClick: React.PropTypes.func.isRequired,
+  },
+
+  componentDidMount() {
+    this.props.assets[0].unlock();
+    this.forceUpdate();
+  },
+
+  render() {
+    let unlockedAssets = [];
+    for (const asset of this.props.assets) {
+      if (asset.unlocked) {
+        unlockedAssets.push(asset);
+      }
+    }
+
+    const buyFxn = this.props.createOnClick;
+
+    return (
+        <div id="store-zone">
+          <h3>Broker</h3>
+          <div id="item-menu">
+            {unlockedAssets.map(function (asset) {
+              return <StoreItem key={asset.name} name={asset.name} price={asset.price} owned={asset.owned} onClick={buyFxn(asset)} />;
```

Comment:
```
return (
  <StoreItem
    key={asset.name}
    name={asset.name}
    price={asset.price}
    owned={asset.owned}
    onClick={buyFxn(asset)}
  />
);
```
in our code base, this is much more common than all on one line.  the parentheses are only needed for multi-line jsxings like this, like all our `render()` returns